### PR TITLE
sql statements: add create binding, drop binding and show bindings

### PR DESCRIPTION
**Pull request by lzmhhh123**
### What is changed, added or deleted? (Required)
add create binding, drop binding and show bindings in sql gramma.

### Which TiDB version(s) do your changes apply to? (Required)

- [X] master (the latest development version)
- [X] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

File: sql-statements/sql-statement-show-bindings.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: SHOW BINDINGS
+summary: TiDB 数据库中 SHOW BINDINGS 的使用概况。
+category: reference
+aliases: ['/docs-cn/dev/reference/sql/statements/show-bindings/']
+---
+
+# SHOW DATABASES
+
+`SHOW BINDINGS` 语句用于显示所有创建过的 sql bind 的相关信息。
```
Comment (bb7133) [inline sql-statements/sql-statement-show-bindings.md:10]:
```suggestion
`SHOW BINDINGS` 语句用于显示所有创建过的 SQL BIND 的相关信息。
```
是否要用大写？

**Comment by bb7133** [comment]
Please resolve the conflicts, rest LGTM.

File: sql-statements/sql-statement-drop-binding.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: DROP BINDING
+summary: TiDB 数据库中 DROP BINDING 的使用概况。
+category: reference
+aliases: ['/docs-cn/dev/reference/sql/statements/drop-binding/']
```
Comment (zimulala) [inline sql-statements/sql-statement-drop-binding.md:5]: This file doesn't need `aliases`. Right？ @yikeke

File: sql-statements/sql-statement-show-bindings.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+title: SHOW BINDINGS
+summary: TiDB 数据库中 SHOW BINDINGS 的使用概况。
+category: reference
+aliases: ['/docs-cn/dev/reference/sql/statements/show-bindings/']
```
Comment (zimulala) [inline sql-statements/sql-statement-show-bindings.md:5]: Ditto.

File: sql-statements/sql-statement-create-binding.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+title: CREATE BINDING
+summary: TiDB 数据库中 CREATE BINDING 的使用概况。
+category: reference
+aliases: ['/docs-cn/dev/reference/sql/statements/create-binding/']
```
Comment (zimulala) [inline sql-statements/sql-statement-create-binding.md:5]: Ditto.

**Comment by lzmhhh123** [comment]
> 请补充 TOC.md

Done.

**Pull request merged**